spec/beta/referral_spec.rb

```diff
@@ -82,4 +82,42 @@
       }.to raise_error(StandardError).with_message('Could not send card details to Stripe, please try again later.')
     end
   end
+
+  describe '.add_payment_method', :authenticate_referral do
```

Comment:
We should assert the status code too, this file might help: https://github.com/EasyPost/easypost-ruby/blob/master/spec/error_spec.rb

Reply:
Sounds good. We don't assert error status codes anywhere else in our tests except for when we're specifically testing error serialization. But we should moving forward.

Reply:
Actually, stratch that. We don't test anything other than the error message in other libraries like Python (https://github.com/EasyPost/easypost-python/blob/6b002bbf1ae1e6fe229205b4baa1efca7ae1e064/tests/test_beta_referral.py#L107). That, combined with the fact that we don't test anything other than the error message in Ruby, I'm going to keep it as is to remain consistent.

Reply:
Gotcha, looks like I am the only one who added these two extra assertions